### PR TITLE
Fixed lane-id param when creating workflow.

### LINKED ISSUE #359: Lane not being set when workflows created
`lane-id` parameter is ignored.

See https://github.com/sul-dlss/workflow-server-rails/blob/master/app/controllers/workflows_controller.rb#L71

### PR DESCRIPTION
**Pull request by justinlittman**
closes #359

## Why was this change made?
So that the lane-id param matches the param passed by workflow client.

## Was the documentation (API, README, DevOpsDocs, wiki, consul, etc.) updated?
No.

## Does this change affect how this application integrates with other services?
If so, please confirm:
- change was tested on stage    and/or
- test added to sul-dlss/infrastructure-integration-test
Tested on stage.

### CODE REVIEW

**Comment by justinlittman** [comment]
image

File: app/services/process_parser.rb [hunk 1]
```diff
@@ -24,8 +24,10 @@ class ProcessParser
 
   ##
   # @param [Nokogiri::XML::Element] process_element
-  def initialize(process_element)
+  # @param [Boolean] provide default as lane_id if no lane_id
+  def initialize(process_element, use_default_lane_id = true)
```
Comment (jcoyne) [inline app/services/process_parser.rb:28]: Would you mind making this a kwarg?  I find this makes the code much easier to read.

File: app/services/process_parser.rb [hunk 2]
```diff
@@ -24,8 +24,10 @@ class ProcessParser
 
   ##
   # @param [Nokogiri::XML::Element] process_element
-  def initialize(process_element)
+  # @param [Boolean] provide default as lane_id if no lane_id
```
Comment (jcoyne) [inline app/services/process_parser.rb:27]: must add the parameter name (e.g. `use_default_lane_id`) here after `[Boolean]`

File: app/services/process_parser.rb (hunk 1 above)
Comment (justinlittman) [reply app/services/process_parser.rb]: I do not mind. Fixed.

File: app/services/process_parser.rb (hunk 2 above)
Comment (justinlittman) [reply app/services/process_parser.rb]: Fixed.

**Pull request merged**